orchestra/frontend/config-overrides.js

```diff
@@ -8,6 +8,11 @@ module.exports = override(
     // the metronome design system has not been compiled before exporting.
     path.resolve('node_modules/@b12/metronome'),
     path.resolve('src/')
-  ])
+  ]),
+  addWebpackExternals({
```

Comment:
This removes the React/ReactRedux packages from the build phase, so that all apps would rely on the global React/ReactRedux package in the global object.

Reply:
add this as an explanatory comment :)